cmd/stack.go

```diff
@@ -136,6 +136,28 @@ func setupStackCommand() *cobra.Command {
 		},
 	}
 
+	dumpCommand := &cobra.Command{
+		Use:   "dump",
+		Short: "Dump stack data for debug purposes",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			output, err := cmd.Flags().GetString(cobraext.StackDumpOutputFlagName)
+			if err != nil {
+				return cobraext.FlagParsingError(err, cobraext.StackDumpOutputFlagName)
+			}
+
+			err = stack.Dump(stack.DumpOptions{
+				Output: output,
+			})
+			if err != nil {
+				return errors.Wrap(err, "dump failed")
+			}
+
+			cmd.Println("Done")
```

Comment:
Consider printing out where the user can find the stack dump directory before "Done".

Reply:
Fixed